README.md

```diff
@@ -190,6 +190,17 @@ If you somehow need to stick to the current version, please make sure to execute
 ```shell
 go mod tidy
 ```
+### Qitmeer gets stuck on synchronization.
```

Comment:
Please add more note to address that the issue can't generally reproduce and is 0.9.x only.

Reply:
updated, adjusted according the latest recommended memory setting
https://github.com/Qitmeer/qitmeer/pull/445/commits/52744e4f9de6fb2d370d9ab0c0491a6110434c08